README.md

```diff
@@ -47,8 +47,21 @@ Install pip for python3
 1. Login the admin panel and create entries in the Navbar Entries section as per your requirement.
 2. Load and use the `navbar` tag if required in any of your custom pages.
 
+### Setting up email templates
+1. Create templates in both text and html formats.
+2. Place them in the directory cited below.
```

Comment:
I think a 'how to send newsletters' will be helpful.

How do we use these templates?